### DISCUSSION TITLE
include comparison with other python SC libs?

### DISCUSSION THREAD

**Discussion by parity3** [agronholm/anyio#412] [Ideas]
It may help (the community) to include a "related projects" area or comparisons area in docs somewhere. I'm not sure how big this list would be realistically.

I found [quattro](https://github.com/Tinche/quattro/issues/1) and although I'm not sure why it exists it may make sense do a comparison with it if that's worthwhile.

**Comment by agronholm** [comment]
IIRC Quattro does not implement level cancellation and does not work with trio. I'm not sure either why it exists.
